irohad/ametsuchi/impl/postgres_wsv_query.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include "ametsuchi/impl/postgres_wsv_query.hpp"
 #include "backend/protobuf/permissions.hpp"
+#include "common/result.hpp"
```

Comment:
This include is presented in postgres_wsv_common.hpp. It is not necessary to have it here

Reply:
It is better not to have transitive includes, so that if we change one header, we won't break stuff